viewing_party/main.py

```diff
@@ -0,0 +1,193 @@
+# WAVE ONE  
+
+def create_movie(title, genre, rating):
+    movie_dict = {
+        "title": title,
+        "genre": genre,
+        "rating": rating
+    }
+    if title == None or genre == None or rating == None:
+        return None 
+    return movie_dict
+
+
+
+def add_to_watched(user_data, movie):
+    user_data["watched"].append(movie)
+    return user_data 
+
+
+
+def add_to_watchlist(user_data, movie):
+    user_data["watchlist"].append(movie)
+    return user_data
+
+
+
+def watch_movie(user_data, title):
+    for movie in user_data["watchlist"]:
+        if movie["title"] == title:
+            user_data["watchlist"].remove(movie)
+            user_data["watched"].append(movie)
+            return user_data
+    return user_data
+
+
+
+# WAVE TWO 
+
+    # PROBLEM ONE 
+    # I need to calculate the total sum of the ratings
+    # Then I need to divide the total sum by the length of the movies in the watched list 
+def get_watched_avg_rating(user_data):
+    sum_counter = 0
+    for movie in user_data["watched"]:
+        sum_counter += movie["rating"] 
+    if len(user_data["watched"]) == 0:
+        return 0.0 
```

Comment:
Consider moving this code to the top of your function to get the edge case situation out of the way before you get into the meat of your function.